benchmarks/distributed/lambda_runtimes/cpp_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Create CMake project
+cmake_minimum_required(VERSION 3.9)
```

Comment:
I think we use 3.12 now as minimum, can you check the root CMakeLists.txt file? Let's try to have the same minimum cmake version everywhere